identity/did/identity.go

```diff
@@ -34,8 +37,15 @@ func NewDIDFromString(address string) DID {
 type Identity interface {
 	// AddKey adds a key to identity contract
 	AddKey(did *DID, key Key) (chan *ethereum.WatchTransaction, error)
+
 	// GetKey return a key from the identity contract
 	GetKey(did *DID, key [32]byte) (*KeyResponse, error)
+
+	// RawExecute calls the execute method on the identity contract
```

Comment:
Should we keep RawExecute public?

Reply:
Is there a use case for it? If not let's make it private.

Reply:
I will make it private.

Reply:
ok!